server/routes/LoggingMiddleware.js

```diff
@@ -0,0 +1,6 @@
+function LoggingMiddleware(req, res, next) {
```

Comment:
What does this do ? Like are we just calling 'next()' for now temporarily as we are not fully done with auth?

Reply:
No, this just logs all the requests to the console. When deploying we can redirect the output to a file or add the option to write logs to a file.